pkg/ccl/backupccl/backup_test.go

```diff
@@ -9726,6 +9720,116 @@ func TestExportRequestBelowGCThresholdOnDataExcludedFromBackup(t *testing.T) {
 	require.NoError(t, err)
 }
 
+// TestExcludeDataFromBackupDoesNotHoldupGC tests that a table marked as
+// `exclude_data_from_backup` and with a protected timestamp record covering it
+// does not holdup GC, since its data is not going to be backed up.
+func TestExcludeDataFromBackupDoesNotHoldupGC(t *testing.T) {
```

Comment:
I solemnly swear to datadriven-ify these tests as part of https://github.com/cockroachdb/cockroach/issues/77129, but to derisk this change I wanted to postpone building out our backup datadriven framework. I have ideas on how to wrap all these GC/SpanConfig operations that I'll work on as part of our test revamp.